examples/springboot/base/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
```

Comment:
error caused by `kustomization.yaml` file is missing.

Reply:
Does this mean the example in the issue never worked?